### ISSUE TITLE
ContextMenu style lacks drop shadow

### ISSUE THREAD

**Issue by chucker** [lepoco/wpfui#712]
### Describe the bug

The default style for `<ContextMenu>` doesn't seem to set a drop shadow (or any other kind of border). However, the style for subseqent submenus does.

### To Reproduce

(I think this is clear from the screenshots.)

### Expected behavior

The visual style should match other menus

### Screenshots

Notice no drop shadow around the menu items:

![image](https://github.com/lepoco/wpfui/assets/20194/67ab8d6d-dff2-4155-9c8d-c82ba365fc7b)

Whereas submenus show one that seems almost too strong?

![image](https://github.com/lepoco/wpfui/assets/20194/747ed5f4-1cc6-4493-b937-b2541aff06a2)

### OS version

Windows 11 22H2 23506.1000

### .NET version

Framework 4.7.2

### WPF-UI NuGet version

2.0.3

### Additional context

I skimmed the style files, and notice that

* `Menu.xaml` sets a drop shadow
* `ContextMenu.xaml` sets a property `HasDropShadow` to `false` (not sure why, and it seems to have no effect), and also doesn't set one either way

**Comment by niels9001** [comment]
With recent updates, a border was added to make sure the contextmenu. @chucker mind testing the latest preview (`preview-10`) to see if that resolves this issue?

**Comment by chucker** [comment]
> to see if that resolves this issue? 

Hmmm, as of 8afb0d9 or `preview-11`, it doesn't seem to. I'm using the Light theme:

        foreach (var uri in new[]
        {
            "pack://application:,,,/Wpf.Ui;component/Resources/Theme/Light.xaml",
            "pack://application:,,,/Wpf.Ui;component/Resources/Wpf.Ui.xaml"
        })
        {
            resources.MergedDictionaries.Add(new ResourceDictionary { Source = new Uri(uri) });
        }

…and applying `UiContextMenu`:

        Style = (Style)FindResource("UiContextMenu");

…and it looks like this:

![image](https://github.com/lepoco/wpfui/assets/20194/dd285d86-9ed2-4531-a1fb-74d68f3b8905)

**Comment by niels9001** [comment]
@chucker Yeah, the only change I'm seeing is that there is a very (subtle) border around the context menu. But agree, ideally this would show a dropshadow as well

**Comment by chucker** [comment]
Ah, you're right — the border is new. That is indeed better than before!

**Comment by chucker** [comment]
Looks like #1472 will resolve this.
